DEVELOPING_OPAL/demos/src/main/scala/org/opalj/ai/ExceptionUsage.scala

```diff
@@ -23,23 +24,22 @@ object ExceptionUsage extends ProjectAnalysisApplication {
     override def doAnalyze(
         theProject:    Project[URL],
         parameters:    Seq[String],
-        isInterrupted: () => Boolean
-    ): BasicReport = {
+        isInterrupted: () => Boolean): BasicReport = {
```

Comment:
This is really ugly, can we avoid it?

Reply:
I think this is the result of the "danglingParentheses" options being set to false, I guess they should be set to true, or is there a reason why they were set to false?